### PR TITLE
input done

### PR DESCRIPTION
**Pull request by theVivek-kumar**
input done please review

### CODE REVIEW

File: imput/input.css [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+    justify-content: center;
+    border: none;
+    padding: .2rem;
+}
+/*textbox*/
+.textarea{
+    width: 100%;
+    height: 6rem;
+    margin: auto;
+}
```
Comment (m26unkwn) [inline imput/input.css:71]: The code structure is good and readable.

Things You Can add.
CSS

1. You can use color variables for your colors
2. You also create variable for font-size and spacing

File: imput/input.html [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+
+
+
+
+
+
+
+</body>
+
+</html>
```
Comment (m26unkwn) [inline imput/input.html:157]: HTML is good but you can also use semantic HTML as it is good for accessibility.

**Review by m26unkwn — APPROVED** [review]
You can do the changes that I have suggested. 
Otherwise, everything is good to go.

File: imput/input.css (hunk 1 above)
Comment (Debarshi95) [reply imput/input.css:71]: I would suggest the same changes

File: imput/input.html (hunk 2 above)
Comment (Debarshi95) [reply imput/input.html:157]: Same here as well

File: imput/input.css [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+
+.radio {
+    margin: 12px;
+    display: grid;
+    grid-template-columns: 1fr 1fr 1fr;
+
+
+}
+.color{
+    color: red;
```
Comment (dev-ashishsethi) [inline imput/input.css:54]: use variable for color

File: imput/input.css [hunk 4]
```diff
@@ -0,0 +1,71 @@
+@import url(/style.css);
+.input-group {
+    margin: 1rem;
+    display: flex;
+    font-size: 1rem;
+    justify-content: center;
+    align-items: center;
+    margin: 10px auto;
```
Comment (dev-ashishsethi) [inline imput/input.css:8]: could have used  rem

**Review by dev-ashishsethi — APPROVED** [review]
do check comments

**Pull request merged**